examples/livekit/README.md

```diff
@@ -26,7 +26,7 @@ In this tutorial we deploy a video room example using [LiveKit's React SDK](http
 
 Let's start with a disclaimer. The LiveKit client example browser must work over a secure HTTPS connection, because [getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#browser_compatibility) is available only in secure contexts. This implies that the client-server signaling connection must be secure too. Unfottunately, self-signed TLS certs [will not work](https://docs.livekit.io/deploy/#domain,-ssl-certificates,-and-load-balancer), so we have to come up with a way to provide our clients with a valid TLS cert. This will have the unfortunate consequence that the majority of the below installation guide will be about securing client connections to LiveKit over TLS; as it turns out, once HTTPS is correctly working integrating LiveKit with STUNner is very simple.
 
-In the below example, STUNner will be installed into the identically named namespace, while LiveKit and the Ingress gateway will live in the default namespace. 
```

Comment:
I hate these "end-of-line" deltas. Can you remove the useless spaces at the end? It'd be best if this PR did not touch the LiveKit example. This is absolutely not critical, just asking

Reply:
Fixed in https://github.com/l7mp/stunner/commit/f14c9110a86f67cad68ded4b8cf6693bb6f45628